### PR TITLE
make robot keyword configurable

### PR DESCRIPTION
**Pull request by bumi**
this makes the keyword for the commands configurable by setting the
OA_BOT_KEYWORD environment varibale. It defaults to 'kredits'.

This allows us to easier run it for different assets.

### CODE REVIEW

File: src/openassets.es6 [hunk 1]
```diff
@@ -29,14 +29,16 @@ module.exports = function(robot) {
   // Address Book
   //
 
+  let robotKeyword = process.env.OA_BOT_KEYWORD || 'kredits';
```
Comment (raucao) [inline src/openassets.es6:32]: Shouldn't we also set the kredits asset ID by default then? If we don't do that, we might as well throw an error if the keyword isn't set I think.

File: src/openassets.es6 [hunk 2]
```diff
@@ -138,17 +140,17 @@ module.exports = function(robot) {
     return parseInt(assetDetails.balance) + parseInt(assetDetails.unconfirmed_balance);
   }
 
-  robot.hear(/kredits show (.+)/i, function(hearResponse) {
+  robot.hear(new RegExp(`${robotKeyword} show (.+)`, 'i'), function(hearResponse) {
     balanceOf(hearResponse.match[1], function(assetDetails) {
       if (assetDetails) {
-        hearResponse.send( hearResponse.match[1] + ' has ' + totalBalanceOfAsset(assetDetails) + ' kredits');
+        hearResponse.send( hearResponse.match[1] + ' has ' + totalBalanceOfAsset(assetDetails) + ' ' + robotKeyword);
```
Comment (raucao) [inline src/openassets.es6:146]: I think this should be "OA_ASSET_NAME" instead. Maybe we should even rename the bot keyword to asset name?

Edit: or even OA_ASSET_CURRENCY_SYMBOL or sth.

File: src/openassets.es6 (hunk 1 above)
Comment (bumi) [inline src/openassets.es6:32]: hmm. ideally I would not make it default and keep it generic. This is just to not break the current setup. 

and actually I think I should also move this on a room level. To be able to configure the assets per room. (for example have devkredits in #kosmos-dev and kredits in #kosmos) 

What do you think? should we move the config then in the hubot-brain?

Comment (raucao) [inline src/openassets.es6:32]: Per room is definitely a good idea. Not sure about moving to brain.

File: src/openassets.es6 (hunk 2 above)
Comment (bumi) [inline src/openassets.es6:146]: agree.

File: src/openassets.es6 (hunk 1 above)
Comment (bumi) [inline src/openassets.es6:32]: how would I make it configurable per room with having the config not in the brain?

**Pull request merged**